api/lib/infrastructure/repositories/knowledge-element-repository.js

```diff
@@ -42,7 +42,7 @@ async function _findByCampaignIdForSharedCampaignParticipationWhere(campaignPart
 
 function _getByUserIdAndLimitDateQuery({ userId, limitDate }) {
   return Bookshelf.knex
-    .select('*', Bookshelf.knex.raw('ROW_NUMBER() OVER (PARTITION BY ?? ORDER BY ?? DESC) AS rank', ['skillId', 'createdAt']))
+    .select('*')
     .from('knowledge-elements')
```

Comment:
```suggestion
   knex('knowledge-elements')
```
Sur bookshelf "vanilla", ça marche d'enlever le select quant tu ramène tout (mais c'est peut être pas assez explicite..)

Reply:
Ah oui oui bien vu, j'avais mis le `select` avant car j'avais plusieurs clauses différentes, ce n'est plus necessaire maintenant. Merci